_pkgdown.yml

```diff
@@ -1,6 +1,5 @@
 url: https://baselinenowcast.epinowcast.org/
 template:
-  package: enwtheme
```

Comment:
this is the standard theme we use across epinowcast projects. I would keep unless there is a good reason to get rid of

Reply:
whoops I might have deleted where it gets imported

Reply:
in the description in the configs section